### DISCUSSION TITLE
Pages hierarchy / app menu

### DISCUSSION THREAD

**Discussion by agolosnichenko** [ToolJet/ToolJet#5505] [Q&A] [answered]
Seems to me that there is no option to make pages hierarchy in [app menu](https://docs.tooljet.com/docs/tutorial/pages).
Also I can't find any menu component in components library. 

So what is the best way to make hierarchical app menu in Tooljet? What are the best practices?

**Comment by withshubh** [comment, marked as answer]
Hey @agolosnichenko - Thanks for creating this discussion.

> So what is the best way to make hierarchical app menu in Tooljet? What are the best practices?

The hierarchy in pages is on the roadmap and it will be available soon.

> Also I can't find any menu component in components library.

Menu component is not available, you can use tabs component instead.
